### PR TITLE
Hide add payment button

### LINKED ISSUE #3960: Do not let users add multiple paypal.me logins
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Context in this PR comment. A user can add multiple paypal.me usernames to their account. 

1. Add paypal.me login
2. Navigate to settings/payments
3. Click on "Add new payment method"
4. Click on PayPal.me
5. Add a Paypal.me account and save
6. Click on add new Payment method again
7. See paypal.me and be able to add another paypal.me login

## Expected Result:
A user should not be able to add multiple paypal.me logins. More specifically, the flow should work as follows: 

1. User adds Paypal.me login.
2. Once paypal.me is added, it is shown in the payment method list
3. A user can tap the existing paypal.me user name to edit it
3. `Add Payment Method` no longer shows paypal.me as an option so long as one exists for the user

## Actual Result:
Describe what actually happened

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platform:
Where is this issue occurring?

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

**Version Number:** has not deployed yet
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/22508290/125133125-5d1f8480-e0d3-11eb-92e4-215132589466.mp4

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by tugbadogan**
cc @roryabraham @kevinksullivan 

### Details
`Add Payment Method` no longer shows paypal.me option if there's an existing Paypal.me username and `Add Payment Method` button is hidden completely if there is no available payment method to add.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/3960

### Tests
Tested by adding and updating Paypal.me username on different platforms.

### QA Steps

- Navigate to settings/payments
- Click on "Add new payment method"
- Click on PayPal.me
- Add a Paypal.me account and save
- Go back to Payment Method List
- Make sure "Add new payment method" button is hidden
- Click and edit existing Paypal.me username

### Tested On

- [X] Web
- [X] Mobile Web
- [X] Desktop
- [X] iOS
- [X] Android

#### Web

https://user-images.githubusercontent.com/3853003/125638660-ecba2340-b29d-479e-ba9d-91ce172323ea.mov

#### Mobile Web

https://user-images.githubusercontent.com/3853003/125638979-3e18c8f7-b919-47e3-9794-ee90fbb71e74.mov

#### Desktop

https://user-images.githubusercontent.com/3853003/125638883-6057096a-faac-4366-baea-92669ea80047.mov

#### iOS

https://user-images.githubusercontent.com/3853003/125639402-8eaa8f96-773d-4b86-9a40-8ad4ec1e2bfd.mov

#### Android

https://user-images.githubusercontent.com/3853003/125639043-bf63feb7-ed2a-43e7-8848-41185e7e1ae1.mov

### CODE REVIEW

File: src/pages/settings/Payments/PaymentsPage.js [hunk 1]
```diff
@@ -48,7 +58,9 @@ class PaymentsPage extends React.Component {
      */
     paymentMethodPressed(nativeEvent, account) {
         if (account) {
-            // TODO: Show the make default/delete popover
```
Comment (tugbadogan) [inline src/pages/settings/Payments/PaymentsPage.js:51]: Are you planning to create separate issue for this TODO item?

**Comment by tugbadogan** [comment]
As I mentioned in the issue, comment these lines to test this PR.
https://github.com/Expensify/Expensify.cash/blob/039b671cc6683372317327fde6dc4f9ecb78818b/src/libs/actions/User.js#L76-L77

File: src/pages/settings/Payments/PaymentsPage.js (hunk 1 above)
Comment (roryabraham) [reply src/pages/settings/Payments/PaymentsPage.js:51]: I'm not sure, but we can remove the TODO

**Review by roryabraham — CHANGES_REQUESTED** [review]
Actually @tugbadogan I was wrong - no API changes are needed. The bug in `getUserDetails` is front-end only. Make the following change to the API call in there and it will work:

```javascript
API.Get({
        returnValueList: 'account, loginList, nameValuePairs',
        nvpNames: [CONST.NVP.BLOCKED_FROM_CONCIERGE, CONST.NVP.PAYPAL_ME_ADDRESS].join(','),
    })
```

**Comment by roryabraham** [comment]
Otherwise, the code looks great! Let's get this easy bug fix added to the PR then this should be good-to-go. Thanks and great work!

**Comment by tugbadogan** [comment]
@roryabraham Thanks for the review. I fixed the API call in `getUserDetails` function. Now it works without any problem.

**Review by roryabraham — APPROVED** [review]

**Pull request merged**